node/libs/crypto/src/bn254/mod.rs

```diff
@@ -0,0 +1,207 @@
+//! BLS signature scheme for the BN254 curve.
```

Comment:
add a disclaimer that it is not secure against side channel timing attacks

Reply:
Added.